lib/common/common.go

```diff
@@ -51,14 +51,10 @@ func ParseDockerURL(arg string) *types.ParsedDockerURL {
 	}
 	indexURL, imageName := SplitReposName(taglessRemote)
 
-	if indexURL == "" && !strings.Contains(imageName, "/") {
+	if indexURL == defaultIndexURL && !strings.Contains(imageName, "/") {
```

Comment:
That's… stupid. It's like leaking implementation detail on all the docker clients. Meh.

Anyway, thanks for making it clear. Maybe a comment would be useful here, because it quite not obvious.